Earthfile

```
@@ -0,0 +1,216 @@
+os-base:
+    FROM condaforge/mambaforge:latest
+
+    ENV DEBIAN_FRONTEND=noninteractive
+    # set locale to utf-8, which is required for some odbc drivers (mysql);
+    ENV LC_ALL=C.UTF-8
+
+    RUN apt-get update && apt-get upgrade -y && \
+        apt-get install -y build-essential zlib1g-dev \
+        wget unixodbc cmake g++ \
+        odbc-postgresql postgresql-client gnupg apt-transport-https && \
```

This should be sufficient.
```suggestion
        apt-get install -y odbc-postgresql postgresql-client gnupg apt-transport-https && \
```

---

@fjetter @bjoernmeier Can we get this in, too?